segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/ChunkedSegmentStorageConfig.java

```diff
@@ -203,4 +203,36 @@
     public static ConfigBuilder<ChunkedSegmentStorageConfig> builder() {
         return new ConfigBuilder<>(COMPONENT_CODE, ChunkedSegmentStorageConfig::new);
     }
+
+    private int getNonNegativeInt(TypedProperties properties, Property<Integer> property) {
```

Comment:
Let's put these on `TypedProperties` and have it throw `InvalidPropertyValueException` just like its counterparts.

Reply:
Agree, this functionality may be useful for other classes, so we can put these methods on a place at which they could be easily reused.